math/mathcore/inc/VectorizedTMath.h

```diff
@@ -0,0 +1,770 @@
+#ifndef ROOT_VectorizedTMath
+#define ROOT_VectorizedTMath
+
+#include "Rtypes.h"
+#include "Math/Types.h"
```

Comment:
The include for `Rtypes.h` is in the else block which executes if veccore is not enabled. Scalar types are sometimes used in functions which require constants and arrays.
Should I put all these include statements after `#ifdef R__HAS_VECCORE` ?

Reply:
Ok, then keep it here. It may make sense later to include unconditionally in `Math/Types.h` in any case, but for now just removing redundancies is fine.